src/gen_swift/templates/module.swift

```diff
@@ -73,9 +73,9 @@ class RNBreezSDK: RCTEventEmitter {
     {% endif -%}
     {%- endfor %}  
     @objc(setLogStream:reject:)
-    func setLogStream(_ resolve: @escaping RCTPromiseResolveBlock, reject: @escaping RCTPromiseRejectBlock) -> Void {
+    func setLogStream(_ filterLevel: LevelFilter = nil, resolve: @escaping RCTPromiseResolveBlock, reject: @escaping RCTPromiseRejectBlock) -> Void {
         do {
-            try BreezSDK.setLogStream(logStream: BreezSDKLogStream())            
+            try BreezSDK.setLogStream(logStream: BreezSDKLogStream(), filterLevel: filterLevel)            
```

Comment:
Convert the optional param
```suggestion
            var levelFilter: LevelFilter? = nil
            if filterLevel != nil {
                levelFilter = try BreezSDKMapper.asLevelFilter(levelFilter: filterLevel!)
            }
            try BreezSDK.setLogStream(logStream: BreezSDKLogStream(), filterLevel: levelFilter)
```